src/studio-home/tabs-section/TabsSection.test.jsx

```diff
@@ -116,39 +125,35 @@ describe('<TabsSection />', () => {
 
       expect(screen.getByText(tabMessages.courseTabErrorMessage.defaultMessage)).toBeVisible();
     });
-  });
 
-  describe('archived tab', () => {
-    it('should switch to Archived tab and render specific archived course details', async () => {
-      render(<RootWrapper />);
+    it('should render pagination when there are courses', async () => {
+      render(<RootWrapper isPaginationCoursesEnabled />);
       axiosMock.onGet(getStudioHomeApiUrl()).reply(200, generateGetStudioHomeDataApiResponse());
-      axiosMock.onGet(courseApiLink).reply(200, generateGetStudioCoursesApiResponse());
-      await executeThunk(fetchStudioHomeData(), store.dispatch);
-
-      const archivedTab = screen.getByText(tabMessages.archivedTabTitle.defaultMessage);
-      fireEvent.click(archivedTab);
+      axiosMock.onGet(courseApiLinkV2).reply(200, generateGetStudioCoursesApiResponseV2());
+      await executeThunk(fetchStudioHomeData('', true, {}, true), store.dispatch);
+      const data = generateGetStudioCoursesApiResponseV2();
+      const coursesLength = data.results.courses.length;
+      const totalItems = data.count;
+      const paginationInfoText = `Showing ${coursesLength} of ${totalItems}`;
 
-      expect(screen.getByText(studioHomeMock.archivedCourses[0].displayName)).toBeVisible();
+      expect(screen.getByText(studioHomeMock.courses[0].displayName)).toBeVisible();
 
-      expect(screen.getByText(
-        `${studioHomeMock.archivedCourses[0].org} / ${studioHomeMock.archivedCourses[0].number} / ${studioHomeMock.archivedCourses[0].run}`,
-      )).toBeVisible();
+      const pagination = screen.getByRole('navigation');
+      const paginationInfo = screen.getByTestId('pagination-info');
+      expect(paginationInfo.textContent).toContain(paginationInfoText);
+      expect(pagination).toBeVisible();
     });
 
-    it('should hide Archived tab when archived courses are empty', async () => {
-      const data = generateGetStudioCoursesApiResponse();
-      data.archivedCourses = [];
-
```

Comment:
this test should be here when the feature flag is off, am I correct? could we add it again?